doc/decisions/remove_elektra_malloc.md

```diff
@@ -0,0 +1,40 @@
+# Remove elektraMalloc() et al.
```

Comment:
A related question is the documentation of the life-time of our objects. Here also improvements of the docu is needed.